modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java

```diff
@@ -575,6 +575,50 @@ public Comment getByExternalReferenceCodeComment(
 			serviceBuilderComment, _commentManager, PortalUtil.getPortal());
 	}
 
+	@Override
+	public Page<Comment> getByExternalReferenceCodeCommentChildCommentsPage(
+			String externalReferenceCode, String commentExternalReferenceCode,
+			String search, Aggregation aggregation, Filter filter,
+			Pagination pagination, Sort[] sorts)
+		throws Exception {
+
+		if (!_objectDefinition.isEnableComments() ||
+			!FeatureFlagManagerUtil.isEnabled(
+				_objectDefinition.getCompanyId(), "LPD-69419")) {
+
+			throw new UnsupportedOperationException();
+		}
+
+		ObjectEntry objectEntry = _getObjectEntry(externalReferenceCode, null);
+
+		com.liferay.portal.kernel.comment.Comment serviceBuilderComment =
+			_fetchComment(
+				ObjectEntry.class.getName(), objectEntry.getId(),
+				commentExternalReferenceCode,
+				_getNonzeroGroupId(objectEntry.getId()));
+
+		if (serviceBuilderComment == null) {
+			throw new NotFoundException();
```

Comment:
Should we include a message with more details, like [this](https://github.com/liferay/liferay-portal/blob/839b6a3083b259deb0b37ba964018a53fcc54036/modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/CommentResourceImpl.java#L654-L664)?
That way, we would also have a similar pattern for comment exceptions.